privacy-security-explainer.md

```diff
@@ -107,6 +126,11 @@ For every call to `XRFrame.getPose()`, the UA must ensure that:
 * The `XRSession.visibility` is set to `visible` 
 * **TODO** address issues [#696](https://github.com/immersive-web/webxr/issues/696) and [#724](https://github.com/immersive-web/webxr/issues/724)
 
+---
```

Comment:
Rather than using the HRs here to visually differentiate the note, it would probably be more consistent to format the note section as a quote (prefix with a ">")

Reply:
agreed, done